rasa/utils/common.py

```diff
@@ -504,3 +504,14 @@ def find_unavailable_packages(package_names: List[Text]) -> Set[Text]:
 def module_path_from_class(clazz: Type) -> Text:
     """Return the module path of an instance's class."""
     return clazz.__module__ + "." + clazz.__name__
+
+
+def get_bool_env_variable(variable_name: str, default_variable_value: bool) -> bool:
+    true_values = ("true", "1")
+    false_values = ("false", "0")
+    value = os.getenv(variable_name, None)
```

Comment:
I think we can define here `default=default_variable_value`, rather than None, then you can remove lines 513-514.

Reply:
I tried that but then it would complicate handling of valid inputs. This way value will only be of type string and that makes it easy to handle.

Reply:
you could do this and remove lines 514 and 515
```
value = os.getenv(variable_name, default=str(default_variable_value))
```